client/src/plus/offer-overview/offer-overview-subscribe/stripe.tsx

```diff
@@ -0,0 +1,4 @@
+import "@stripe/stripe-js";
```

Comment:
I wonder if it's better to use an async `import()` rather than `React.lazy`. I'm unsure how something like:
```js
export default function Stripe() {
  (async () => {
    await import("@stripe/stripe-js");
  })()
  return null
}
```
behaves with re-renders. I guess you could wrap it in a useCallback, but then is it any less "reacty" than just using React.lazy?